tests/runtime/engine/runner.py

```diff
@@ -307,7 +307,8 @@ def run_test(test):
 
         if test.cleanup:
             try:
-                cleanup = subprocess.run(test.cleanup, shell=True, capture_output=True, text=True)
+                cleanup = subprocess.run(test.cleanup, shell=True, stderr=subprocess.PIPE,
+                                         stdout=subprocess.PIPE, text=True)
```

Comment:
Isn't `text=True` breaking python 3.6 installations.

Reply:
surprisingly no. So, I kept it same. In such case what is your view "text or universal_newlines" ?

Reply:
Not sticking with "text" option, although its passing. Replacing with universal_newlines=True